### PR TITLE
Added a download to search history as soon as the search is started. …

### LINKED ISSUE #298: History Section:  Section not showing up for the first retrival request
While performing a search for a file, the entire History section including the requested veteran name and ID information is not showing up for the first veteran on the eFolder Express landing page.  The History section does show up when I perform a second file retrieval request.   

### Reproduction Steps
1. Enter Vet ID number in the search.
1. Click the Search button.
1. Click on the Start Retrieving eFolder button.
1. While file is being retrieved, click on the eFolder Express logo.  
1. The landing page is displayed.  There is no History section showing on the eFolder landing page nor does the first file being requested show.

Refer to images below in comments

### PR DESCRIPTION
**Pull request by nemodjuric**
…Fixed test for recent downloads.
To test: Do a search and before it finishes click on eFolder Express to get back to the Search page. The search should be present in History section.
fixes #298

### CODE REVIEW

**Comment by amprokop** [comment]
I don't quite understand this fix— would you mind adding a screenshot and some more explanation about what was wrong, @nemodjuric? Thanks!

**Comment by nemodjuric** [comment]
The issue was referenced in the description #298. Screenshots are there as well...
The problem was that once you do a search and go back to the Search page, it wouldn't show that download since it was not completed.

File: app/controllers/downloads_controller.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@ def downloads
   end
 
   def recent_downloads
-    @recent_downloads ||= downloads.where(status: [3, 4, 5, 6])
+    @recent_downloads ||= downloads.where(status: [0, 1, 2, 3, 4, 5, 6])
```
Comment (shanear) [inline app/controllers/downloads_controller.rb]: We didn't include these for a reason. Because the download has not started in these states.

**Comment by amprokop** [comment]
@nemodjuric — sure, I see that there's a description in the issue of the problem from a user-facing perspective. In the PR for a fix like this, I'd like to see an explanation of what was technically incorrect about the code and how your changeset fixes it. That way, whoever is reviewing can get some quick context and evaluate it better.

**Comment by shanear** [comment]
this issue has been closed

**Pull request closed without merging**